test/functions.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+require '../functions.php';
+
+use PHPUnit\Framework\TestCase;
+
+class functions extends TestCase
+{
+    public function testSuccessDisplayCards_1()
+    {
+        $input = [['name' => 'England', 'place' => 'Lulworth', 'landmark' => 'Durdle Door', 'activity' => 'Walking', 'food' => 'sea food']];
+        $actualOutput = displayCards($input);
+        $this->assertIsString($actualOutput);
+
+    }
+
+    public function testSuccessDisplayCards_2()
+    {
+        $input = [['name' => 'England', 'place' => 'Lulworth', 'landmark' => 'Durdle Door', 'activity' => 'Walking', 'food' => 'sea food']];
+        $expectedOutput = '<div class="countryCard"><div class="name"><p>England</p></div><div class="items"><p>Place: Lulworth</p><p>Landmark: Durdle Door</p><p>Activity: Walking</p><p>Food: sea food</p></div></div>';
+        $actualOutput = displayCards($input);
+        $this->assertEquals($expectedOutput, $actualOutput);
+    }
+
+    public function testFailureDisplayCards()
```

Comment:
it would be good here to add a comment that explains what is incorrect about the input (i.e. why it's a failure) and what graceful handling you've chosen (it returns an empty string if the input is invalid)